test/utils/audit.go

```diff
@@ -101,6 +104,10 @@ func parseAuditLine(line string, version schema.GroupVersion) (AuditEvent, error
 	if e.RequestObject != nil {
 		event.RequestObject = true
 	}
+	if e.ImpersonatedUser != nil {
+		event.ImpersonatedUser = e.ImpersonatedUser.Username
+		event.ImpersonatedGroups = strings.Join(e.ImpersonatedUser.Groups, ",")
```

Comment:
Hmm, this is going to be flaky if there is more than 1 group. I think you should just sort the groups before the join.

Reply:
Good catch. 👍